source/_integrations/growatt_server.markdown

```diff
@@ -17,13 +17,13 @@ This will log into your Growatt account and grab the first "Plant", after which
 
 ## Configuration
 
-Add the following to your `configuration.yaml` file:
+You can set this integration up using the integration page.
+Or you can add the following to your `configuration.yaml` file:
```

Comment:
When a configuration flow is introduced, the YAML configuration should be deprecated and thus be removed from the documentation.